diff --git a/node_modules/@remix-run/react/dist/scroll-restoration.js b/node_modules/@remix-run/react/dist/scroll-restoration.js
index 89077d8..3681f10 100644
--- a/node_modules/@remix-run/react/dist/scroll-restoration.js
+++ b/node_modules/@remix-run/react/dist/scroll-restoration.js
@@ -90,7 +90,7 @@ function ScrollRestoration({
       let positions = JSON.parse(sessionStorage.getItem(STORAGE_KEY) || "{}");
       let storedY = positions[restoreKey || window.history.state.key];
       if (typeof storedY === "number") {
-        window.scrollTo(0, storedY);
+        window.scrollTo({left: 0, top: storedY, behavior: 'instant'});
       }
     } catch (error) {
       console.error(error);
